test/Makefile

```diff
@@ -97,6 +97,7 @@ QUIET=@
 export RESULTS_DIR=test_results
 export MODEL
 export REQUIRED_ARGS=
```

Comment:
BTW, working with exported ENV parameters is really annoying when reading a log failure and not being able to rerun a command.